### PR TITLE
ci: release please

### PR DESCRIPTION
**Pull request by skyerus**
Creates PR for a release & updates the version found in README.md

### CODE REVIEW

File: release-please-config.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "bootstrap-sha": "95df901b104dadc8902117c87d2e412d78d6a620",
```
Comment (beeme1mr) [inline release-please-config.json:2]: Is this a valid sha?

File: .github/workflows/release-please.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    runs-on: ubuntu-latest
+
+    # Release-please creates a PR that tracks all changes
+    steps:
+      - uses: google-github-actions/release-please-action@v3
+        id: release
+        with:
+          command: manifest
+          token: ${{secrets.GITHUB_TOKEN}}
+          default-branch: main
```
Comment (beeme1mr) [inline .github/workflows/release-please.yml:18]: Should the release flow be included here?

File: release-please-config.json (hunk 1 above)
Comment (skyerus) [reply release-please-config.json:2]: No, I admit I stole this from another repo.. I tested that the process worked in my forked repo
I've just tried to run
```
release-please bootstrap \
  --token=$GITHUB_TOKEN \
  --repo-url=skyerus/open-feature-operator \
  --release-type=go
```
to generate the sha but I'm getting a 404. I might need to ask for some help on this later

File: .github/workflows/release-please.yml (hunk 2 above)
Comment (skyerus) [reply .github/workflows/release-please.yml:18]: No, the release flow happens when the release tag is created whereas this flow occurs every time something new is pushed to main

File: release-please-config.json (hunk 1 above)
Comment (beeme1mr) [reply release-please-config.json:2]: Just do a `git log` and grab a sha. Since haven't used conventional commits in the past, any commit sha should work.

Comment (beeme1mr) [reply release-please-config.json:2]: Speaking of conventional commits, you should add the commit linter here. We'll always want to update the merging options to use the PR title as the commit message.

**Comment by toddbaert** [comment]
You might want to add a blurb in the readme about this.

**Comment by skyerus** [comment]
> You might want to add a blurb in the readme about this.

@toddbaert about what specifically, the use of please release as a release workflow or that we're auto updating the versions?

**Comment by toddbaert** [comment]
> the use of please release as a release workflow

Ya, was thinking something like this: https://github.com/open-feature/node-sdk-contrib#releases

File: release-please-config.json (hunk 1 above)
Comment (skyerus) [reply release-please-config.json]: Done

**Comment by skyerus** [comment]
> > the use of please release as a release workflow
> 
> Ya, was thinking something like this: https://github.com/open-feature/node-sdk-contrib#releases

Yoink, done! Thanks

**Review by beeme1mr — APPROVED** [review]

**Comment by beeme1mr** [comment]
@AlexsJones, are you okay with this PR? In this case, release please is used to bump the version number on the readme.

**Comment by AlexsJones** [comment]
I am very okay with it
But I want to understand

```
<!---x-release-please-start-version-->
```
kubectl create namespace open-feature-operator-system
kubectl apply -f https://github.com/open-feature/open-feature-operator/releases/download/v0.0.9/certificate.yaml
kubectl apply -f https://github.com/open-feature/open-feature-operator/releases/download/v0.0.9/release.yaml
```
<!---x-release-please-end-->
```

Does this auto detect the semver and substitute or does the entire block get replaced - if so where is the replacement text kept?

**Review by AlexsJones — APPROVED** [review]

**Comment by skyerus** [comment]
> I am very okay with it But I want to understand
> 
> ```
> 
> ```
> 
> kubectl create namespace open-feature-operator-system kubectl apply -f https://github.com/open-feature/open-feature-operator/releases/download/v0.0.9/certificate.yaml kubectl apply -f https://github.com/open-feature/open-feature-operator/releases/download/v0.0.9/release.yaml
> 
> ```
> 
> ```
> 
> Does this auto detect the semver and substitute or does the entire block get replaced - if so where is the replacement text kept?

It auto detects the semver and substitutes it

**Comment by AlexsJones** [comment]
> cts the semver and substitutes it

Rock n roll

**Pull request merged**